store.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Karma</title>
+        <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    </head>
+    <body>
+              <header class="header">
+                    <img class="karma_logo" src="img/karma-logo.svg" alt="karma logo"> 
+                        <!--navabar-->
+                        <nav> 
+                            <ul class="nav_list">
+                                <li class="nav_item">
+                                    <a class="nav_link" id= "meet_karma"href=""><strong>Meet Karma</strong></a>
+                                </li>
+                                <li class="nav_item">
+                                    <a class="nav_link" href="">How It Works</a>
+                                </li>
+                                <li>
+                                    <a class="nav_link" href="">Store</a>
+                                </li class="nav_item">
+                                <li>
+                                    <a class="nav_link" href="">Blog</a>
+                                </li class="nav_item">
+                                <li>
+                                    <a class="nav_link" href="">Help</a>
+                                </li class="nav_item"> 
+                                <li>
+                                    <a class="nav_link" href="">Login</a>
+                                </li>
+                            </ul>
+                        </nav>
+            </header>
+            <main>
+                <div class="wrapper-form">
+                    <form>
+                        <h2 class="orange-header">Order your Karma wifi device today!</h2>
+                        <div class="row">
```

Comment:
Consider using [fieldset](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) instead of generic `div`s

Reply:
I change it, It's better.